sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ExpressionEvalHelper.scala

```diff
@@ -93,6 +95,40 @@ trait ExpressionEvalHelper extends GeneratorDrivenPropertyChecks {
     }
   }
 
+  protected def checkExceptionInExpression[T <: Throwable](
+      expression: => Expression,
+      inputRow: InternalRow,
+      expectedErrMsg: String)(
+      implicit classTag: ClassTag[T]): Unit = {
+    val clazz = classTag.runtimeClass
+    val serializer = new JavaSerializer(new SparkConf()).newInstance
```

Comment:
This code is also present in `checkEvaluation` can you factor it out into a common method?